hs_core/models.py

```diff
@@ -3962,7 +4166,7 @@ def update(self, metadata, user):
                 self.update_repeatable_element(element_name=element_name, metadata=metadata,
                                                property_name="funding_agencies")
 
-    def get_xml(self, pretty_print=True, include_format_elements=True):
+    def get_xml_legacy(self, pretty_print=True, include_format_elements=True):
```

Comment:
Can this be replaced by the new serializer implementation HydroPrettyXMLSerializer? Does the fact that you changed the func name to get_xml_legacy mean it will be replaced subsequently in a separate PR?

Reply:
All resource types outside of Composite/Collections still use the old way of creating rdf/xml.  I didn't bother converting them because they are deprecated and will eventually be replaced by aggregation implementations.